src/ADAL.PCL.Desktop/WindowsFormsWebAuthenticationDialogBase.cs

```diff
@@ -152,9 +156,16 @@ protected virtual void WebBrowserNavigatingHandler(object sender, WebBrowserNavi
 
             // check if the url scheme is of type browser://
             // this means we need to launch external browser
-            if (!e.Cancel && e.Url.Scheme.Equals("browser", StringComparison.CurrentCultureIgnoreCase))
+            if (!e.Cancel && e.Url.Scheme.Equals(BrowserScheme, StringComparison.OrdinalIgnoreCase))
             {
-                Process.Start(e.Url.AbsoluteUri.Replace("browser://", "https://"));
+                // Build the HTTPS URL for launching with an external browser
+                var httpsUrlBuilder = new UriBuilder(e.Url)
```

Comment:
Using `System.UriBuilder` to change the scheme from `browser://` to `https://` instead of string-replacement (which was previously not doing so ignoring case).